apps/channels/forms.py

```diff
@@ -21,3 +21,11 @@ class TelegramChannelForm(forms.Form):
 
 class WhatsappChannelForm(forms.Form):
     number = forms.CharField(label="Number", max_length=100)
+
+
+class FacebookChannelForm(forms.Form):
```

Comment:
You could add the webhook URL into the form as a disabled field with a help text giving some context:

```
webook_url = forms.Charfield(label="Webhook URL", disabled=True,
    help_text="Copy this into the Facebook setup form.")
```

Reply:
https://github.com/dimagi/open-chat-studio/pull/115/commits/20913cded1b4f1f385497242a7365ab47f6b602e